docs/dotnet-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope .NET Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> These configuration settings can also be provided via CLI flags. Run `scope-run --help` for more information.
+
+
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=False
```

Comment:
don't you need to export? Or inline this with your command. Otherwise this does nothing, right?

Reply:
yes, but every platform (windows, Mac, linux,...) has a different way to set Env variables, so I thought that showing g export explicitly was not clear, also GitHub Action has a different way to set them

Reply:
I'd show only `SCOPE_AUTO_INSTRUMENT=false`. Not always needed to use commands, e.g. GitHub Actions.

Reply:
That variable doesn't exist in the dotnet agent anymore, the name is `SCOPE_INSTRUMENTATION_ENABLED` to follow the same pattern as the `scope.yml`:
```yaml
scope:
    instrumentation:
        enabled: true
```

Reply:
maybe we need to review all env vars, but should match the `scope.yml` file

Reply:
Changed for .net

Reply:
Env vars should be standard for all agents isn't?